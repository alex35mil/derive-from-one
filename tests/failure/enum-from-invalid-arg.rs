use derive_from_one::FromOne;

#[derive(FromOne)]
pub enum T {
    #[from(ignore)]
    A(bool),
    B(&'static str),
}

fn main() {}
