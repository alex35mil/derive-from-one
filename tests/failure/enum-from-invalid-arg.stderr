error: Expected `#[from(skip)]`
 --> tests/failure/enum-from-invalid-arg.rs:5:12
  |
5 |     #[from(ignore)]
  |            ^^^^^^
